russian/gusli/vodka/vodka.md

```diff
@@ -1,7 +1,7 @@
 Наконец-то вы добралсь до полторалитровой мутной самогонки, выгнанной судя по всему из зефира.
 Оторвав кусок зефира от стены, вы делаете несколько глотков прямо из сосуда и занюхиваете зефиром.
 "Что-то не то" - думаете вы. 
-Быстрыми проворными движениями вы придаете зефиру форму шпротины и с жадностью заедаетевыпитое.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.